config/install/core.entity_view_display.node.audience.full.yml

```diff
@@ -227,7 +227,7 @@ third_party_settings:
               label_display: '0'
               provider: menu_block
               context_mapping: {  }
-              follow: true
```

Comment:
These following changes are unwanted. I'm not sure why they are included in this pull request.